cicd/pyinstaller/product_cli.spec

```diff
@@ -66,6 +66,22 @@ cortxcli = Analysis([cc_path + '/cli/cortxcli.py'],
              cipher=block_cipher,
              noarchive=False)
 
+cli_setup = Analysis([cc_path + '/cli/conf/cli_setup.py'],
+             pathex=['/usr/lib/python3.6/site-packages/'],
```

Comment:
@avrnv We are planning to remove pyinstaller. Hence `cicd/pyinstaller` directory wont be in picture.
For adding `cli_setup` as binary, please check https://github.com/Seagate/cortx-manager/pull/406/files#diff-2344324d1ead713aeedd540e4d327417154fae8fb76567f006879e42dc498735R282

Just copy cli_setup.py into $DIST/cli/lib